### ISSUE TITLE
[FEATURE REQUEST] Define cds settings in package.json of this package

### ISSUE THREAD

**Issue by gregorwolf** [Soontao/cds-mysql#486]
Hello @Soontao,

great to see that there is a MySQL evolving for CAP. Right now you instruct users to add:

```JSON
{
  "requires": {
    "db": {
      "kind": "mysql"
    },
    "kinds": {
      "mysql": {
        "impl": "cds-mysql"
      }
    }
  }
}
```

to the package.json. You could avoid this by following the approach of @cap-js/postgres. Check out: 

https://github.com/cap-js/cds-dbs/blob/main/postgres/package.json#L46

Maybe @johannes-vogel has some details if there is any additional documentation if you run into issues.

**Comment by Soontao** [comment]
Thanks, will try the new approach!

**Comment by Soontao** [comment]
released with `cds-mysql@7.4.0`
